### ISSUE TITLE
Mirror window doesn't open

### ISSUE THREAD

**Issue by Grellenord** [Genymobile/scrcpy#3650]
Environment

    OS: Win10
    scrcpy version: 1.25
    installation method: prebuilt package for Windows
    device model: SAMSUNG A32 4G

Description: 

scrcpy.exe seems to start normally. phone wakes up, but no window is shown on the desktop; process ends after.

```
c:\tools\scrcpy>scrcpy -V verbose
scrcpy 1.25 <https://github.com/Genymobile/scrcpy>
DEBUG: ADB device found:
DEBUG:     -->   (usb)           RF8RC07ZEYR            device  SM_A325F
DEBUG: Device serial: RF8RC07ZEYR
DEBUG: Using server (portable): c:\tools\scrcpy\scrcpy-server
c:\tools\scrcpy\scrcpy-server: 1 file pushed, 0 skipped. 28.4 MB/s (42151 bytes in 0.001s)
[server] INFO: Device: samsung SM-A325F (Android 12)
[server] DEBUG: Using encoder: 'c2.mtk.avc.encoder'
DEBUG: Server connected
DEBUG: Starting controller thread
DEBUG: Starting receiver thread
[server] DEBUG: Controller stopped

c:\tools\scrcpy>[server] DEBUG: Screen streaming stopped
[server] DEBUG: Device message sender stopped
```

**Comment by rom1v** [comment]
Try with another encoder: https://github.com/Genymobile/scrcpy#encoder

**Comment by Grellenord** [comment]
Tried all from the following list:

`[server] ERROR:     scrcpy --encoder 'c2.mtk.avc.encoder'
[server] ERROR:     scrcpy --encoder 'OMX.MTK.VIDEO.ENCODER.AVC'
[server] ERROR:     scrcpy --encoder 'c2.android.avc.encoder'
[server] ERROR:     scrcpy --encoder 'OMX.google.h264.encoder'`

Result is always the same.

**Comment by rom1v** [comment]
Does `screenrecord` work?

```
adb shell screenrecord /sdcard/file.mp4
adb pull /sdcard/file.mp4
```

**Comment by Grellenord** [comment]
Yes, without any problem.

**Comment by rom1v** [comment]
```
scrcpy --no-display --record=file.mkv
```

Does it work?

**Comment by Grellenord** [comment]
Just fine.

**Comment by rom1v** [comment]
OK, so everything works fine except opening a window on the computer.

Looks like #3431 or #3561 but was a problem on Linux.

Does `scrcpy --render-driver=opengl` work?

Could you test the previous version (with an older SDL version): https://github.com/Genymobile/scrcpy/releases/tag/v1.24

**Comment by Grellenord** [comment]
Using `--render-driver=opengl` led to exactly the same result as before. 

Version 1.24 was the one I used originally, I downloaded 1.25 only after encountering this problem :) so yes, I have the same result there.

**Comment by Grellenord** [comment]
Works just fine out-of-the-box with the same device on my PC at home. It really could be caused by some overzealous antivirus or whatever...

**Comment by VituminX** [comment]
@Grellenord I have the exact same issue with identical debug messages. I can get it to work on my personal PC but my work PC wont load the mirror window.

I'm also guessing some sort of corporate control

**Comment by ottercynical** [comment]
Non-functional app?  Same problem on my home pc.  How do I fix this?
